### ISSUE TITLE
I train normally, but there is no validation process

### ISSUE THREAD

**Issue by JY00002** [open-mmlab/mmrotate#7]
I train normally, but there is no validation process

i use this shell:
python -m torch.distributed.launch --nproc_per_node 4 train.py ./configs/r3det/r3det_tiny_r50_fpn_1x_dota_oc.py --launcher pytorch --gpus 4

got:

2022-02-19 09:47:48,227 - mmrotate - INFO - workflow: [('train', 1)], max: 12 epochs
2022-02-19 09:47:48,227 - mmrotate - INFO - Checkpoints will be saved to /home/ai/yumo/ship_det/mmrotate-main/work_dirs/r3det_tiny_r50_fpn_1x_dota_oc by HardDiskBackend.
2022-02-19 09:48:13,565 - mmcv - INFO - Reducer buckets have been rebuilt in this iteration.
2022-02-19 09:48:32,743 - mmrotate - INFO - Epoch [1][50/100]	lr: 9.967e-04, eta: 0:17:03, time: 0.890, data_time: 0.494, memory: 3223, s0.loss_cls: 1.0393, s0.loss_bbox: 0.3072, sr0.loss_cls: 0.9379, sr0.loss_bbox: 0.3254, loss: 2.6098, grad_norm: 4.0230
2022-02-19 09:48:52,601 - mmrotate - INFO - Exp name: r3det_tiny_r50_fpn_1x_dota_oc.py
2022-02-19 09:48:52,602 - mmrotate - INFO - Epoch [1][100/100]	lr: 1.163e-03, eta: 0:11:48, time: 0.397, data_time: 0.006, memory: 3224, s0.loss_cls: 0.5890, s0.loss_bbox: 0.2449, sr0.loss_cls: 0.4457, sr0.loss_bbox: 0.2588, loss: 1.5385, grad_norm: 4.9931
2022-02-19 09:49:37,768 - mmrotate - INFO - Epoch [2][50/100]	lr: 1.330e-03, eta: 0:12:40, time: 0.885, data_time: 0.490, memory: 3224, s0.loss_cls: 0.4540, s0.loss_bbox: 0.2310, sr0.loss_cls: 0.3741, sr0.loss_bbox: 0.2568, loss: 1.3159, grad_norm: 4.4678
2022-02-19 09:49:57,411 - mmrotate - INFO - Exp name: r3det_tiny_r50_fpn_1x_dota_oc.py
2022-02-19 09:49:57,411 - mmrotate - INFO - Epoch [2][100/100]	lr: 1.497e-03, eta: 0:10:41, time: 0.393, data_time: 0.006, memory: 3224, s0.loss_cls: 0.3914, s0.loss_bbox: 0.2394, sr0.loss_cls: 0.3382, sr0.loss_bbox: 0.2556, loss: 1.2246, grad_norm: 4.6190
2022-02-19 09:50:42,743 - mmrotate - INFO - Epoch [3][50/100]	lr: 1.663e-03, eta: 0:10:56, time: 0.893, data_time: 0.492, memory: 3224, s0.loss_cls: 0.3672, s0.loss_bbox: 0.2198, sr0.loss_cls: 0.3200, sr0.loss_bbox: 0.2618, loss: 1.1688, grad_norm: 4.4310

**Comment by yangxue0827** [comment]
You need to modify the hyperparameter (evaluation) in your configs, default setting is evaluation = dict(interval=12, metric='mAP') in https://raw.githubusercontent.com/open-mmlab/mmrotate/main/configs/_base_/schedules/schedule_1x.py
